azure-pipelines.yml

```diff
@@ -12,6 +19,20 @@ resources:
     name: OpenAstronomy/azure-pipelines-templates
     ref: master
 
+trigger:
```

Comment:
I don't have the brain space to get my head around these, can we leave them out for now?

Reply:
This just configure what azure will trigger its CI on.

Reply:
👍